t/issue/1439.t

```diff
@@ -0,0 +1,17 @@
+package main;
```

Comment:
Fixed it in follow up commit.

Reply:
Ah! I added the dummy package and the trailing `1;` because perlcritic complained. I didn't realize it wanted a shebang for unit tests. That makes way more sense.